tests/test_neuro_oracle.py

```
@@ -0,0 +1,405 @@
+        "threshold": [1.0, 1.0],
+        "decay": [0.5, 0.5],
+        "refractory_steps": 0,
+        "reset": "subtract",
+        "dt_ms": 1.0,
+        "action_labels": ["hold", "advance"],
+    }
+    model.update(overrides)
+    return model
+
+
+def _stimulus(steps=4, channels=2):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_stimulus`

(ANN202)